llvm/test/tools/llvm-dwarfdump/verify-no-file.yaml

```
@@ -0,0 +1,356 @@
+# RUN: rm -rf %t && mkdir -p %t
```

No need to create a directory:
```suggestion
 # RUN: yaml2obj %s -o %t.o
 # RUN: llvm-dwarfdump --debug-line --verify %t.o 2>&1 | FileCheck %s
```